content/about/student_stories.rst

```diff
@@ -0,0 +1,68 @@
+Student Stories
+===============
+:slug: student-stories
+:title: Student Stories
+:menu: Student Experience, Student Stories, 3
+:img: OSLWorkers2_adjusted.jpeg
+
+Our Student Stories are a special collection of experiences and thoughts from
+the students themselves. Check out what our student employees find exciting and
+worth taking some time to tell you about.
+
+Amanda Kelner on Graduating
+---------------------------
+
+As of September 8th, my time here at OSU will officially come to an end. As sad
+as I'll be to leave my life here in Corvallis and as nervewracking it is to
+enter the real world, I realized recently I've spent the last seventeen years of
+my life in school and I'm ready to break free!
+
+I'll be leaving with both a degree in English and in music performance, both of
+which have taught me so much not just about their respective fields, but about
+growing up and how to work with what you've got.
+
+While I did not have the skills or the privilage to be a student developer or
+systems administrator, I feel that my time here at the OSL has been incredibly
+valuable. I've learned so much here that I would never have gained from a class.
+The challenges I've faced and the experiences I've had taught me diligence,
+patience, and flexibility. I firmly believe that my time here at the OSL has
+shaped me into a person who is unafraid and excited to leave college.
+
+I'm more than ready to start my new life, but I will miss the friends I've made
+and the life I've created here at OSU. The best of luck to my peers and go
+beavs!
+
+Cody Holliday on Why we should stop using C
+-------------------------------------------
+
+Programming languages are a touchy topic in Computer Science. In certain crowds
+even mentioning a language will elicit groans and eye-rolling. Conversely, there
+are crowds that will only use certain languages for all projects.
+
+These people have lost sight of the fact that programming languages are tools.
+Languages have certain problem sets that they're really good at and some not so  
+much. If you were to ask me to do some complex math or signal processing I would
+point you to MATLAB. Would I use MATLAB for developing a GUI? Not in a million years.
+So why do we choose C? Well, C is efficient since it's practically one step above
+assembly and with GCC it compiles down to a small executable. This is great for 
```

Comment:
Expand on this, maybe switch GCC with "modern compilers" as well since clang does really really well on program size as well